web-calculator.js

```diff
@@ -0,0 +1,25 @@
+var a, b, result;
+function setValues() {
+    a = Number(document.getElementById('a').value);
+    b = Number(document.getElementById('b').value);
+}
+function ADD() {
+    setValues();
+    result = a + b;
+    alert(a + " + " + b + " = " + result);
+}
+function SUBTRACT() {
+    setValues();
+    result = a - b;
+    alert(a + " - " + b + " = " + result);
+}
+function DIVIDE() {
+    setValues();
+    result = a / b;
+    alert(a + " / " + b + " = " + result);
+}
+function MULTIPLY() {
+    setValues();
+    result = a * b;
+    alert(a + " * " + b + " = " + result);
+}
```

Comment:
You've got a lot of repetitive code here since you are doing different functions. They could be handled with a single function. Also notice the `.join()` trick to get the spaces, that's a fun one.

```JavaScript
function calculate(operator) {
  var num1 = Number(document.getElementById('a').value)
  var num2 = Number(document.getElementById('b').value)
  var result

  switch(operator) {
    case '+':
      result = num1 + num2
      break
    case '-':
      result = num1 - num2
      break
    case '*':
      result = num1 * num2
      break
    case '/':
      result = num1 / num2
      break
    default:
      result = 'Unknown operator'
  }
  
  alert([a, operator, b, '=', result].join(' ))
}
```